frontend/app/services/application-status-service.server.ts

```diff
@@ -100,5 +101,77 @@ function createApplicationStatusService() {
     return statusResponse.BenefitApplication.BenefitApplicationStatus[0].ReferenceDataID;
   }
 
-  return { getStatusId };
+  /**
+   * @returns the status id of a dental application given the application code and client information without the SIN
+   */
+  async function getStatusIdWithoutSin({ applicationCode, firstName, lastName, dateOfBirth }: GetStatusIdWithoutSinArgs) {
+    log.debug('Fetching status id of dental application for application code [%s]', applicationCode);
+    log.trace('Fetching status id of dental application for application code [%s], first name [%s], lastname [%s], date of birth [%s]', applicationCode, firstName, lastName, dateOfBirth);
+    const instrumentationService = getInstrumentationService();
+
+    getAuditService().audit('application-status.post', { userId: 'anonymous' });
+
+    const url = new URL(`${INTEROP_STATUS_CHECK_API_BASE_URI ?? INTEROP_API_BASE_URI}/dental-care/application/v1/status_fnlndob`);
+    const statusRequest = {
+      BenefitApplication: {
+        Applicant: {
+          PersonName: [
+            {
+              PersonGivenName: [firstName],
+              PersonSurName: lastName,
+            },
+          ],
+          PersonBirthDate: {
+            date: dateOfBirth,
+          },
+          ClientIdentification: [
+            {
+              IdentificationID: applicationCode,
+            },
+          ],
+        },
+      },
+    };
+
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Ocp-Apim-Subscription-Key': INTEROP_STATUS_CHECK_API_SUBSCRIPTION_KEY ?? INTEROP_API_SUBSCRIPTION_KEY,
+      },
+      body: JSON.stringify(statusRequest),
+    });
+
+    if (!response.ok) {
+      instrumentationService.countHttpStatus('http.client.interop-api.status.posts', response.status);
```

Comment:
In order to distinguish metrics for calls between `/status` vs. `/status_fnlndob`:

```suggestion
      instrumentationService.countHttpStatus('http.client.interop-api.status-fnlndob.posts', response.status);
```


Also, if we move this line above the `if (!response.ok)`, we can remove the `countHttpStatus('http.client.interop-api.status.posts', 200)` line below.